pkg/operator/operator.go

```diff
@@ -443,6 +443,30 @@ func New(
 		klog.Info("Metrics Server enabled")
 	}
 
+	// csrMetricsServerController runs a controller that requests a client TLS certificate
+	// for Metrics Server. This certificate is used to authenticate against the
+	// /metrics endpoint of the targets.
+	csrMetricsServerController, err := csr.NewClientCertificateController(
+		csr.ClientCertOption{
+			SecretNamespace: "openshift-monitoring",
+			SecretName:      "metrics-server-client-certs",
+			AdditionalAnnotations: certrotation.AdditionalAnnotations{
+				JiraComponent: "Monitoring",
+			},
+		},
+		csrOption,
```

Comment:
Haven't checked all the changes, shouldn't we have a new `csrOption`?

Reply:
aah right, good catch